src/file.hpp

```diff
@@ -557,11 +557,7 @@ std::vector<LineData*> readFile(const char* filePath)
                     }
                 }
             } else {
-                if (line->token.empty()) {
-                    delete line;
-                } else {
-                    result.push_back(line);
-                }
+                result.push_back(line);
```

Comment:
A fatal bug in which deleting a line with a detected error in the initial file parsing stage will not display the error information and continue assembly **“as if the line were not there.”**